code/modules/clothing/suits/ego_gear/special.dm

```diff
@@ -24,3 +24,52 @@
 	desc = "This armor is strong to pale, check it's defenses to see!"
 	icon_state = "freshman"
 	armor = list(RED_DAMAGE = -40, WHITE_DAMAGE = 0, BLACK_DAMAGE = 0, PALE_DAMAGE = 40)
+
+//Nihil Event rewards
+/obj/item/clothing/suit/armor/ego_gear/despair/nihil
+	name = "meaningless despair"
+	desc = "As with sorrow, perhaps sharing the burden will blunt the edge."
+	icon_state = "despair_nihil"
+	armor = list(RED_DAMAGE = 60, WHITE_DAMAGE = 80, BLACK_DAMAGE = 50, PALE_DAMAGE = 90) // 280
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80,
+							PRUDENCE_ATTRIBUTE = 80,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 100
+							)
+
+/obj/item/clothing/suit/armor/ego_gear/hatred/nihil
+	name = "pointless hate"
+	desc = "She vowed to love everything in the world, but all that was left was a collapsing heart."
+	icon_state = "hate"
+	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 60, BLACK_DAMAGE = 90, PALE_DAMAGE = 60) // 280
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80,
+							PRUDENCE_ATTRIBUTE = 100,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 80
+							)
+
+/obj/item/clothing/suit/armor/ego_gear/blind_rage/nihil
+	name = "shameless wrath"
+	desc = "The Servant was betrayed after abandoning her principles and ingenuously trusting someone with her whole heart."
+	icon_state = "wrath"
+	armor = list(RED_DAMAGE = 90, WHITE_DAMAGE = 20, BLACK_DAMAGE = 90, PALE_DAMAGE = 80) // 280
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80,
+							PRUDENCE_ATTRIBUTE = 80,
+							TEMPERANCE_ATTRIBUTE = 100,
+							JUSTICE_ATTRIBUTE = 80
+							)
+
+/obj/item/clothing/suit/armor/ego_gear/goldrush/nihil
+	name = "worthless greed"
+	desc = "Now, only visceral greed remains."
+	icon_state = "greed"
+	armor = list(RED_DAMAGE = 90, WHITE_DAMAGE = 60, BLACK_DAMAGE = 70, PALE_DAMAGE = 60) // 280
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 100,
+							PRUDENCE_ATTRIBUTE = 80,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 80
+							)
```

Comment:
What's the point of these being subtypes of the original? I believe almost everything they had is changed and this only serves to cause the issue of you needing to override these lest they be hit with the card and changed into themselves.

Reply:
Good point, what was I thinking?